scripts/analyze-comp-file.py

```
@@ -58,7 +58,7 @@
 
 total_square_vars_no_comp = 0
 
-for ppt, comp_set_sizes_at_ppt in ppt_info_dict.items():  # noqa: B007 PERF102
+for ppt, comp_set_sizes_at_ppt in ppt_info_dict.items():  # ruff:ignore[unused-loop-control-variable, incorrect-dict-iterator]
```

Ruff (0.15.21)

[warning] Loop control variable `ppt` not used within loop body

Rename unused `ppt` to `_ppt`

(B007)